brtconfig.py

```diff
@@ -81,7 +81,7 @@ def get_docker_labels(self):
     From config file get the labels that should be added to the image
     """
 
-    config = module_framework.get_correct_config()
```

Comment:
Hi @sgallagher, I think support old version is not good idea (not now before 1.0 release)
Sorry for this change, we've discussed it with Irina and we've found that "get_correct_config" is the worst name what I could choose. so thats why this change. we'll ship also in fedora new version to rename this method this week.